go/test/endtoend/cluster/mysqlctl_process.go

```diff
@@ -63,6 +63,7 @@ func (mysqlctl *MysqlctlProcess) InitDb() (err error) {
 		args = append(args, "--")
 	}
 
+	//TODO: Remove underscore(_) flags in v25, replace them with dashed(-) notation
 	args = append(args, "--init_db_sql_file", mysqlctl.InitDBFile)
```

Comment:
Is the reason we can't change this yet because we run this same code for upgrade / downgrade tests from this version? Or do we then run the code from the older version for this test setup too?

Reply:
This is a pretty common pattern in the code, I haven't looked at these particular tests. Quite a few in https://github.com/vitessio/vitess/pull/18280 as well, etc.

Reply:
I guess we run this endtoend code also with the older version, so maybe it's what we have to live with until we branch off v23 and then we clean it up for v24? 

Regardless, it's something we can defer then for a later PR as well.